### PR TITLE
[WIP] PrivacyTool's Recommendations

### PR DESCRIPTION
**Pull request by tommytran732**
Dunno if we should have this post, but I do wanna see what other thinks.

PrivacyTools is indeed a malicious actor and Burung Hantu does not actually care whether he things he recommends is actually good or not. I think he is just doing it for the affiliate and donation money. It might be a good idea to have all of the bad recommendations laid out fewer people would fall for them.

That said, this is WIP, I do think we should elaborate a bit more on why these are bad. Thinks like deterministic password manager could be talked about in detail in a separate post.

### CODE REVIEW

**Comment by jermanuts** [comment]
many typos like the word appropriate https://deploy-preview-35--privsec-dev.netlify.app/knowledge/privacytools.io/#oo-shutup10. Will this blog get merged? Also looking forward for windows guide. Thanks

**Comment by tommytran732** [comment]
> many typos like the word appropriate https://deploy-preview-35--privsec-dev.netlify.app/knowledge/privacytools.io/#oo-shutup10. Will this blog get merged? Also looking forward for windows guide. Thanks

I dunno, I wanna see of others think iit should be on the site or not. I was told that we probably should wait till later for something like this.

File: content/knowledge/PrivacyTools.io.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+PrivacyTools.io is a fairly popular website recommending software and providers for the privacy communities. However, the website lacks any sort of quality control, recommending many products without technical merits or with severe vulnerabilities, and ending up harming user privacy. This post will go over a non-exhaustive list of bad recommendations from PrivacyTools.io.
+
+## Web Browsers
+
+### Duckduckgo
+Duckduckgo Browser on Android is a [Webview based browser](https://github.com/duckduckgo/Android/issues/1837#issuecomment-1093044254). It [does not support Site Isolation](https://chromium.googlesource.com/chromium/src/+/main/docs/process_model_and_site_isolation.md). This is in contrast with Standalone or Trichrome browsers which support this feature and come preinstalled out of the box with most Android-based operating systems.
+
+### Onion Browser
+The Onion Browser leaks the actual IP address of the user via WebRTC. This issue has been reported in Juanuart 2018 and still has not been fixed as of this writing (July 2022). You can read more about this [here](https://github.com/OnionBrowser/OnionBrowser/issues/117).
```
Comment (v6ak) [inline content/knowledge/PrivacyTools.io.md:16]: ”Juanuart“ should probably have been “January”.

File: content/knowledge/PrivacyTools.io.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+### LineageOS
+LineageOS substantially weakens the baseline security of Android. It does not support verified boot even on devices which support it, does not ship firmware updates on the vast majority of builds, uses userdebug builds, and so on.
+
+## "Encrypted DNS Clients for Improved Internet Privacy"
+Encrypted DNS [does not](https://madaidans-insecurities.github.io/encrypted-dns.html) provide privacy.
+
+## Anonymizing Networks
+
+### Zeronet
+Zeronet [deos not](https://zeronet.io/docs/faq/#how-to-use-zeronet-with-tor) provide anonimity. It is about as private as the Bittorrent network, which means it is not very private on its own at all. Instead, the user has to opt into using the Tor Network in conjuction with it for anonimity.
```
Comment (v6ak) [inline content/knowledge/PrivacyTools.io.md:73]: “anonimity” —> “anonymity”

File: content/knowledge/PrivacyTools.io.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+### Retroshare
+Same as Zeronet. Unless it is used with Tor or I2P, it does not provide anonimity. The "circle of trust" can see each other's IP addresses.
```
Comment (v6ak) [inline content/knowledge/PrivacyTools.io.md:76]: “anonimity” —> “anonymity”

File: content/knowledge/PrivacyTools.io.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+### IPFS
+IPFS [does not](https://docs.ipfs.tech/concepts/privacy-and-encryption/) provide anonimity. It is designed as a public network, much like BitTorrent.
```
Comment (v6ak) [inline content/knowledge/PrivacyTools.io.md:79]: “anonimity” —> “anonymity”

File: content/knowledge/PrivacyTools.io.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+### Spectre
+This is yet another deterministic password manager like Lesspass. However, unlike Lesspass, it is also stateless, which makes it even less flexible.
+
+## Productivity & Collaboration Tools
+
+### Etherpad
+Etherpad does not support end to end encryption, and does not even have password authentication out of the box. There is nothing "private" about it.
+
+### OnlyOffice
+OnlyOffice in itself is not an entirely terrible application; however, it has worse security properties than Microsoft Office and Apple iWork. On Windows, it does not have [MDAG integration](https://www.onlyoffice.com/) like Microsoft Office. On macOS, it is not [sandboxed](https://developer.apple.com/documentation/security/app_sandbox) like Apple iWork or Microsoft Office. OnlyOffice should only be recommended with caveats attached to it.
```
Comment (v6ak) [inline content/knowledge/PrivacyTools.io.md:46]: The MDAG integration link should probably point somewhere else.

**Review by v6ak — COMMENTED** [review]
I've briefly reviewed it. It looks good, expect some typos and one bad link.

**Pull request closed without merging**